### PR TITLE
ENG-34465 push to ecr

### PR DESCRIPTION
**Pull request by tiennguyenskedulo**
The original Dockerfile is taken from [here](https://github.com/Skedulo/elasticserver/blob/dev/dev/database/Dockerfile).

### CODE REVIEW

**Comment by jamesbassett** [comment]
I think you'll need the init scripts as well right? i.e. https://github.com/Skedulo/elasticserver/tree/dev/dev/database/docker-entrypoint-initdb.d

**Comment by jamesbassett** [comment]
You'll definitely want to test this in a few APIs (like Elasticserver and Auth Service) to prove that it works. Did you want to create a few PRs with this change applied, or did you want us to?

**Comment by tiennguyenskedulo** [comment]
@jamesbassett  I have created the PRS to test it in [Elasticserver](https://github.com/Skedulo/elasticserver/pull/899) and [Auth Service](https://github.com/Skedulo/auth-service/pull/291).

File: Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ RUN apt-get update \
         build-essential \
         pkg-config \
         curl \
-        postgresql-server-dev-14=$PG_VERSION
+        postgresql-server-dev-14
```
Comment (JoeSkedulo) [inline Dockerfile:17]: Did this not work with `PG_VERSION`?

**Review by JoeSkedulo — APPROVED** [review]
Looks good, test branches look fine

File: Dockerfile (hunk 1 above)
Comment (tiennguyenskedulo) [reply Dockerfile:17]: I actually not sure what is the expected value for this variable and where it is defined when looking in elasticserver project.

Comment (JoeSkedulo) [reply Dockerfile:17]: I think it will be coming from the base image `postgres:14.4-bullseye`

Comment (tiennguyenskedulo) [reply Dockerfile:17]: I think it said `Version '14.4-1.pgdg110+1' for 'postgresql-server-dev-14' was not found` https://app.circleci.com/pipelines/github/Skedulo/dev-database/2/workflows/4c13bc7e-f96a-4cae-aabc-1872355419ab/jobs/1

Comment (JoeSkedulo) [reply Dockerfile:17]: Okay no worries, we can leave it out

**Review by jamesbassett — APPROVED** [review]
Looks good to me too Tien - nice work 👍

**Pull request merged**